### ISSUE TITLE
Keywords are not working

### ISSUE THREAD

**Issue by Jassob** [ayberkt/chalmers-msc-thesis-template#3]
For some reason, I can't get `\keywords` command to work correctly.. 

With this example, I expect the keywords in the abstract to be "Some, random, words" but instead some strange default value ("keywords") pop ups instead.  

```tex
\input{template/template}

\title{The Thesis Title}
\author{John Doe}

\keywords{Some, random, words}

\begin{document}

\begin{abstract}
  \lipsum[1-3]
\end{abstract}

\end{document}
```

Note that this is with the following patch applied, as the keywords otherwise are hard-coded to "foo, bar"

```diff
diff --git a/settings.tex b/settings.tex
index 31f438f..0b355e7 100644
--- a/settings.tex
+++ b/settings.tex
@@ -62,7 +62,7 @@
   }
   {% KEYWORDS (MAXIMUM 10 WORDS)
    \vfill
-   Keywords: foo, bar
+   Keywords: \@keywords

    \newpage	% Create empty back of side
    \thispagestyle{empty}
```

The result is available [here](https://github.com/ayberkt/chalmers-msc-thesis-template/files/4187844/main.pdf).

**Comment by ayberkt** [comment]
@Jassob there is something weird going on with variable names involving `@`. Apparently, the fact that the variable contains an `@` is the reason why we need the `makeatletter/makeatother` nonsense. I resolved the issue in #4 simply by picking a name that does not contain `@`.

Can you confirm that #4 works for you as well?

**Comment by Jassob** [comment]
Yup, it works perfectly! Well done!
